docs/api/browser-window.md

```diff
@@ -774,1044 +291,102 @@ Objects created with `new BrowserWindow` have the following properties:
-
-**Note:** If the menu bar is auto-hide, users can still bring up the menu bar by pressing the single `Alt` key.
-
-####  `win.kiosk`
-
-A `Boolean` property that determines whether the window is in kiosk mode.
-
-#### `win.documentEdited` _macOS_
-
-A `Boolean` property that specifies whether the window’s document has been edited.
-
-The icon in title bar will become gray when set to `true`.
-
-#### `win.representedFilename` _macOS_
-
-A `String` property that determines the pathname of the file the window represents,
-and the icon of the file will show in window's title bar.
-
-#### `win.title`
-
-A `String` property that determines the title of the native window.
-
-**Note:** The title of the web page can be different from the title of the native window.
-
-#### `win.minimizable`
-
-A `Boolean` property that determines whether the window can be manually minimized by user.
-
-On Linux the setter is a no-op, although the getter returns `true`.
-
-#### `win.maximizable`
-
-A `Boolean` property that determines whether the window can be manually maximized by user.
-
-On Linux the setter is a no-op, although the getter returns `true`.
-
-#### `win.fullScreenable`
-
-A `Boolean` property that determines whether the maximize/zoom window button toggles fullscreen mode or
-maximizes the window.
-
-#### `win.resizable`
-
-A `Boolean` property that determines whether the window can be manually resized by user.
-
-#### `win.closable`
-
-A `Boolean` property that determines whether the window can be manually closed by user.
-
-On Linux the setter is a no-op, although the getter returns `true`.
-
-#### `win.movable`
-
-A `Boolean` property that determines Whether the window can be moved by user.
-
-On Linux the setter is a no-op, although the getter returns `true`.
-
-#### `win.excludedFromShownWindowsMenu` _macOS_
-
-A `Boolean` property that determines whether the window is excluded from the application’s Windows menu. `false` by default.
-
-```js
-const win = new BrowserWindow({ height: 600, width: 600 })
-
-const template = [
-  {
-    role: 'windowmenu'
-  }
-]
-
-win.excludedFromShownWindowsMenu = true
-
-const menu = Menu.buildFromTemplate(template)
-Menu.setApplicationMenu(menu)
-```
-
-#### `win.accessibleTitle`
-
-A `String` property that defines an alternative title provided only to
-accessibility tools such as screen readers. This string is not directly
-visible to users.
-
 ### Instance Methods
 
 Objects created with `new BrowserWindow` have the following instance methods:
 
 **Note:** Some methods are only available on specific operating systems and are
 labeled as such.
 
-#### `win.destroy()`
-
-Force closing the window, the `unload` and `beforeunload` event won't be emitted
-for the web page, and `close` event will also not be emitted
-for this window, but it guarantees the `closed` event will be emitted.
+#### `browserWindow.focusOnWebView()`
 
-#### `win.close()`
+#### `browserWindow.blurWebView()`
```

Comment:
Are these new methods? They're lacking documentation.

Reply:
They are not new, they did not have description in the original documentation. I can add descriptions in new PRs.